src/components/share-button/share-button.component.ts

```diff
@@ -105,24 +73,25 @@ export class ShareButtonComponent implements AfterViewInit {
                 this.popUpClosed.emit(this.button.provider);
             }
         }, 200);
-    }
+    };
 
+    shareButton() {
```

Comment:
Following the **"avoid DOM manipulation"** comment, this should be in the template of `share-button`